media/js/firefox/new.js

```diff
@@ -170,6 +170,8 @@
                     $scene1.css('visibility', 'hidden');
                     $thankYou.focus();
                 }, 500);
+                // Added measurements fire only when the download is fired
+                spring_campaign_measurement();
```

Comment:
This line can still be called twice - click CTA, click back button, click CTA again. Not a very likely scenario, but would be nice to put a check in before adding the pixel a second time.